$OpenBSD: patch-mozilla_storage_mozStorageService_cpp,v 1.1 2020/10/15 06:53:09 landry Exp $

dont check for SQLITE_SECURE_DELETE at runtime, we disable it

Index: mozilla/storage/mozStorageService.cpp
--- mozilla/storage/mozStorageService.cpp.orig
+++ mozilla/storage/mozStorageService.cpp
@@ -201,7 +201,6 @@ Service::getSingleton()
   // or newer.  Our configure check ensures we are using a new enough version
   // at compile time.
   if (SQLITE_VERSION_NUMBER > ::sqlite3_libversion_number() ||
-      !::sqlite3_compileoption_used("SQLITE_SECURE_DELETE") ||
       !::sqlite3_compileoption_used("SQLITE_THREADSAFE=1") ||
       !::sqlite3_compileoption_used("SQLITE_ENABLE_FTS3") ||
       !::sqlite3_compileoption_used("SQLITE_ENABLE_UNLOCK_NOTIFY") ||
